tern/command_lib/snippets.yml

```diff
@@ -11,15 +11,20 @@ apt-get:
     - 'install' # subcommand to install
   remove:
     - 'purge' # subcommand to remove a package
+    ignore: # list of subcommands that don't add or remove packages
+    - 'update' 
+  packages: 'dpkg' # refer to base.yml's method of collection
+
+apt:
+  install: 'install' # subcommand to install
+  remove: 'purge' # subcommand to remove a package
```

Comment:
You could just copy exactly what is under `apt-get` and you should be fine.

Reply:
Yeah, actually I did that but I think `prettier` did this😅
Updated again.